HunterPie/MainWindow.xaml.cs

```diff
@@ -1,132 +1,133 @@
+                {
+                    e.Cancel = true;
+                    return;
+                }
+            }
+
+            AsyncHelper.RunSync(PoogieApi.EndSession);
+
+            base.OnClosing(e);
+        }
+
+        private void OnInitialized(object sender, EventArgs e)
+        {
+            InitializerManager.InitializeGUI();
+            
+            InitializeDebugWidgets();
+                       
+            SetupTrayIcon();
+            SetupMainNavigator();
+        }
+
+        private void OnKeyDown(object sender, KeyEventArgs e)
+        {
+            if (e.KeyboardDevice.IsKeyDown(Key.LeftCtrl) && e.KeyboardDevice.IsKeyDown(Key.R))
+                App.Restart();
+        }
+
+        private void InitializeDebugWidgets() => DebugWidgets.MockIfNeeded();
+
+        private void SetupTrayIcon()
+        {
+            TrayService.AddDoubleClickHandler(OnTrayShowClick);
+
+            TrayService.AddItem("Show")
+                .Click += OnTrayShowClick;
+
+            TrayService.AddItem("Close")
+                .Click += OnTrayCloseClick;
+        }
+
+        private void SetupMainNavigator()
+        {
+            DoubleAnimation shrinkAnimation = new DoubleAnimation(1.5, 1, TimeSpan.FromMilliseconds(200))
+            {
+                EasingFunction = new QuarticEase()
+            };
+            DoubleAnimation opacityAnimation = new DoubleAnimation(0, 1, TimeSpan.FromMilliseconds(500))
+            {
+                EasingFunction = new SineEase()
+            };
+            MainHost.Instance.PropertyChanged += (_, __) =>
+            {
+                PART_ContentPresenter.BeginAnimation(FrameworkElement.OpacityProperty, opacityAnimation);
+                PART_ContentPresenter.RenderTransform.BeginAnimation(ScaleTransform.ScaleXProperty, shrinkAnimation);
+                PART_ContentPresenter.RenderTransform.BeginAnimation(ScaleTransform.ScaleYProperty, shrinkAnimation);
+            };
+        }
+
+        private void OnTrayShowClick(object sender, EventArgs e)
+        {
+            Show();
+            WindowState = WindowState.Normal;
+            Focus();
+        }
+
+        private void OnTrayCloseClick(object sender, EventArgs e) => Close();
+
+        private void OnStartGameClick(object sender, EventArgs e) => Steam.RunGameBy(ClientConfig.Config.Client.DefaultGameType);
+
+        private void OnMouseDown(object sender, MouseButtonEventArgs e)
+        {
+            if (IsMouseOverElement(e, PART_HeaderBar))
+                return;
+
+            if (!IsMouseOverElement(e, PART_NotificationsPanel))
+                PART_HeaderBar.ViewModel.IsNotificationsToggled = false;
+        }
+
+        private bool IsMouseOverElement(MouseEventArgs args, FrameworkElement element)
+        {
+            Point points = args.GetPosition(element);
+            HitTestResult result = VisualTreeHelper.HitTest(element, points);
+
+            return result != null;
+        }
+    }
+}

```

Comment:
Not really a complaint but for some reason your git is diff'ing the whole files and it makes it a bit hard to see what were the actual changes. Make sure your IDE isn't formatting the code by itself or adding spaces/tabs

Reply:
sorry. idk, what makes few files being strange... :(
I only used VS editor under the same condition and uploaded. but I'll check options out.